applications/nrf_desktop/sample.yaml

```diff
@@ -2,36 +2,36 @@ sample:
   name: nRF52 Desktop
   description: BLE HID device reference design
 tests:
-  test:
+  applications.nrf_desktop:
```

Comment:
@wiminordic , does this change affect as anyhow?

Reply:
It could affect sanity-check, @thst-nordic do we need to update anything in the CI because of those changes?

Reply:
This change was needed by sanitycheck. It was throwing:
`nrf/applications/nrf_desktop: can't load (skipping): bad test name 'test' in ...  Tests should reference the category and subsystem with a dot as a separator.`